### ISSUE TITLE
Private Chat

### ISSUE THREAD

**Issue by Vinod1144** [AKouki/SignalR-Chat#8]
Hi @AKouki 

Instead of creating room always need to chat with Online member.

can we do this in this code?

Thanks.

**Comment by AKouki** [comment]
You can send private message using the /private(username) text command. But yes, you must be in room.

I am planing next days to implement the private chat in separate window...

**Comment by Vinod1144** [comment]
Hi @AKouki 

Thanks for the prompt response,
is it possible to do it in upcoming two days?
i can pay the extra amount for the work.
connect me over the WhatsApp for further discussion : - WhatsApp number : - +91 - 9601258730

Thanks looking forward to working with you.

**Comment by AKouki** [comment]
It is not required. This is an open-source project.

I will try as soon as posible to implement it, or atleast an early version

**Comment by Vinod1144** [comment]
yes, i understand but i need to deliver it to my end customer and i need this feature, that is why i'm asking.

but no worries it's upto you.

Thanks.

**Comment by AKouki** [comment]
You just want an real time private chat. You don't want to save anything and come back later right? Because that requires the change of database schema and few others things which i don't know if i can help you in that case.

**Comment by Vinod1144** [comment]
for now on quick update bases you can do it without database driven, that's fine.

**Comment by AKouki** [comment]
I have uploaded an very simple/basic solution for direct messages under **feature/dm** branch.

The complexity of the project has increased so i must completly rewrite/redesign it. But, for now i don't have the time.

Please fork/copy the feature/dm branch and let me know so i can delete this branch.

**Comment by Vinod1144** [comment]
Okay sure give me sometime i will check and let you know
